backend/src/main/java/quarano/department/web/TrackedCaseRepresentations.java

```diff
@@ -207,13 +207,13 @@ private ErrorsDto validateAfterEnrollment(TrackedCaseDto source, ErrorsDto error
 	@Retention(RetentionPolicy.RUNTIME)
 	@Target(ElementType.PARAMETER)
 	@Validated({ Default.class, ValidationGroups.Index.class })
-	static @interface ValidatedIndexCase {
```

Comment:
Bitte in den Commit Kommentar erwähnen warum static entfernt wurde (ich nehme an weil static ohne default implementierung ohnehin sinnlos ist?)

Reply:
Ok, war für mich ein no-brainer weil mir mein IntelliJ Ultimate mitgeteilt hat, dass das `static` Schlüsselwort redundant ist.